sdk/servicebus/Azure.Messaging.ServiceBus/src/Amqp/AmqpMessageConverter.cs

```diff
@@ -580,7 +577,7 @@ public static AmqpMap GetRuleDescriptionMap(RuleProperties description)
             return ruleDescriptionMap;
         }
 
-        public static RuleProperties GetRuleDescription(AmqpRuleDescriptionCodec amqpDescription)
+        public virtual RuleProperties GetRuleDescription(AmqpRuleDescriptionCodec amqpDescription)
```

Comment:
Can we move this and GetFilter into AmqpRuleManager? I think it didn't belong in this class.